src/main.rs

```diff
@@ -53,6 +49,61 @@ impl EventHandler for Handler {
         info!("Logged into Discord as {}", ready.user.name)
     }
 
+    async fn reaction_add(&self, ctx: Context, reaction: Reaction) {
+        if let None = reaction.guild_id {
+            return;
+        }
+
+        if let Ok(chan) = env::var("LOGGING_CHANNEL") {
+            let channel_id = ChannelId(chan.parse::<u64>().unwrap()); 
+
+            let escalate_emoji = vec!["❗", "‼️", "⁉️", "❕"];
+            match reaction.emoji {
+                ReactionType::Unicode(ref tmp) => {
+                    if escalate_emoji.contains(&tmp.as_str()) {
+                        let message = reaction.channel_id.message(&ctx, reaction.message_id).await.unwrap();
+                        let mut files: Vec<Vec<u8>> = Vec::new();
+                        let mut urls: String = String::from(" ");
+                        for a in &message.attachments {
+                            if a.size > 8388608 {
+                               urls.push_str(&a.url);     
+                            } else {
+                                files.push(a.download().await.unwrap());
+                            }
+                        }
```

Comment:
Looks like you started implementing attaching files but never finished it, since neither `files` nor `urls` are referenced again. It even goes through the effort of downloading attachments, so it could be doing a lot of work for nothing. This should either be disabled or the attachments should be sent with the forwarded message

Reply:
sounds good one sec